src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -63,99 +78,6 @@ internal class RazorLanguageServerClient : ILanguageClient, ILanguageClientCusto
         remove { }
     }
 
-    [ImportingConstructor]
-    public RazorLanguageServerClient(
-        RazorCustomMessageTarget customTarget,
-        RazorLanguageClientMiddleLayer middleLayer,
-        LSPRequestInvoker requestInvoker,
-        ProjectConfigurationFilePathStore projectConfigurationFilePathStore,
-        RazorLanguageServerLogHubLoggerProviderFactory logHubLoggerProviderFactory,
-        [Import(AllowDefault = true)] IOutputWindowLogger? outputWindowLogger,
-        LanguageServerFeatureOptions languageServerFeatureOptions,
-        ProjectSnapshotManagerDispatcher projectSnapshotManagerDispatcher,
-        ILanguageClientBroker languageClientBroker,
-        ILanguageServiceBroker2 languageServiceBroker,
-        ITelemetryReporter telemetryReporter,
-        IClientSettingsManager clientSettingsManager,
-        ILspServerActivationTracker lspServerActivationTracker,
-        [Import(AllowDefault = true)] VisualStudioHostServicesProvider? vsHostWorkspaceServicesProvider)
```

Comment:
The change is a bit hidden due to the move to primary constructors, but it's the removal of the `AllowDefault`s and question marks.